internal/common/testing/container/container.go

```diff
@@ -36,7 +37,7 @@ type ID string
 // Containers holds the set of running containers during the test.
 type Containers struct {
 	cli               client.CommonAPIClient
-	runningContainers []ID
+	runningContainers map[ID]Container
```

Comment:
Does "Container" need to be public?

Reply:
this is inherited from https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/306, but I think it's preferable to be exported since it's the return type of `StartImage` and a parameter for the proposed `RemoveContainer`